README.md

```diff
@@ -1,23 +1,279 @@
+use elkodon_bb_posix::signal::SignalHandler;
+
+fn main() {
+    let event_name = ServiceName::new(b"MyEventName").unwrap();
+
+    let event = zero_copy::Service::new(&event_name)
+        .event()
+        .open_or_create()
+        .expect("failed to create/open event");
+
+    let notifier = event
+        .notifier()
+        .create()
+        .expect("failed to create notifier");
+
+    let mut counter: u64 = 0;
+    while !SignalHandler::was_ctrl_c_pressed() {
+        counter += 1;
+        notifier
+            .notify_with_custom_trigger_id(EventId::new(counter))
+            .expect("failed to trigger event");
+
+        println!("Trigger event with id {} ...", counter);
+        std::thread::sleep(std::time::Duration::from_secs(1));
+    }
+}
+```
+
+**listener.rs**
+
+```rust
+use elkodon::prelude::*;
+use elkodon_bb_posix::signal::SignalHandler;
+
+fn main() {
+    let event_name = ServiceName::new(b"MyEventName").unwrap();
+
+    let event = zero_copy::Service::new(&event_name)
+        .event()
+        .open_or_create()
+        .expect("failed to create/open event");
+
+    let mut listener = event
+        .listener()
+        .create()
+        .expect("failed to create listener");
+
+    while !SignalHandler::was_ctrl_c_pressed() {
+        for event_id in listener
+            .timed_wait(std::time::Duration::from_secs(1))
+            .expect("failed to wait on listener")
+        {
+            println!("event was triggered with id: {:?}", event_id);
+        }
+    }
+}
+```
+
+This example is a simplified version of the
+[event example](examples/examples/event/). You can
+execute it by opening two terminals and calling:
+
+**Terminal 1:**
+
+```sh
+cargo run --example event_notifier
+```
+
+**Terminal 2:**
+
+```sh
+cargo run --example event_listener
+```
+
 ## Supported Platforms
 
-### Target Platforms
+The support levels can be adjusted when required.
+
+| Operating System | Current Support Level | Target Support Level |
+|------------------|:---------------------:|---------------------:|
+| Android          | tier 3                | tier 1               |
+| FreeBSD          | tier 1                | tier 1               |
+| FreeRTOS         | tier 3                | tier 2               |
+| iOS              | tier 3                | tier 2               |
+| Linux (x86_64)   | tier 1                | tier 1               |
+| Linux (aarch64)  | tier 1                | tier 1               |
+| Linux (32-bit)   | tier 3                | tier 1               |
+| Mac OS           | tier 3                | tier 2               |
+| QNX              | tier 3                | tier 1               |
+| WatchOS          | tier 3                | tier 2               |
+| Windows          | tier 2                | tier 2               |
+
+- **tier 1** - All safety and security features are working.
+- **tier 2** - Works with a restricted security and safety feature set.
+- **tier 3** - Not yet working but planned.
```

Comment:
I think I was a bit imprecise in my last comment. I did not meant to remove the `State` column but to extend the `support` column. I'm fine with keeping the columns as they are now but I would replace some (not all) of the tier 3 with just `planned`. Limux (32-bit) would be tier 3 and QNX just planned. Currently this gives the impression that there is already code written specifically for those platforms but overall it is not yet working.